cmd/csi/init/builder.go

```diff
@@ -69,22 +69,16 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 
 		signalHandler := ctrl.SetupSignalHandler()
 
-		access, err := metadata.NewAccess(signalHandler, dtcsi.MetadataAccessPath)
+		// new schema
+		conn, err := metadata.NewDBAccess(dtcsi.MetadataAccessPath)
```

Comment:
wdyt if we rename it back to NewAccess and may be create follow up to rename it properly, it will reduce amount of changes.

Reply:
Done